recipes-noop/halservices/gst-drm-plugins-rpi.bb

```diff
@@ -0,0 +1,16 @@
+SUMMARY = "No Operation Recipe for gstreamer drm plugins"
+LICENSE = "Apache-2.0"
```

Comment:
Usually gstreamer plugins are licensed as LGPLv2.1 in RDK although the ones here may be "Proprietary".  Either way, apache-2.0 is not idea, even for dummy entries.  Can you use either "LGPLv2"+ or "Proprietary" with a matching LICENSE file (perhaps from COREBASE)?